core/src/main/java/org/apache/accumulo/core/file/rfile/RFile.java

```diff
@@ -746,50 +746,45 @@ public boolean hasTop() {
     @Override
     public void next() throws IOException {
```

Comment:
A bunch of these changes are meant to reduce the call stack size and make the jit's job easier. I don't remember if the jit has trouble with optimizing this call in particular, or whether that has changed since the original tests.